### PR TITLE
Test new membership propositions

### PR DESCRIPTION
**Pull request by joelochlann**
This is a test of 3 different variants of a new membership proposition. Three thrashers to display on :

picture 181
picture 186
picture 187

And three landing pages:

screencapture-m-thegulocal-com-commercial-guardian-explore-1458839675438
screencapture-m-thegulocal-com-commercial-guardian-experience-1458839694702
screencapture-m-thegulocal-com-commercial-guardian-explain-1458839836924

@Calanthe 
- [x] Resolve conflicts
- [x] Rename a few things to make them less general, as per @Calanthe's comments
- [x] Display on home (UK) front, uk news front, and politics front
- [ ] Refine styling of thrashers
- [ ] Optimise/remove images as per @Calanthe's comments

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/membership.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            }
+
+            if (config.page.contentType === 'Article') {
+                return $('.fc-container').first();
+            }
+
+            return null;
+        }
+
+        function test(name, description, linkHref) {
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/experiments/tests/membership.js]: This function's name is too general. What do you want to test exactly?

File: static/src/stylesheets/content.scss [hunk 2]
```diff
@@ -118,6 +118,7 @@
    ========================================================================== */
 
 @import 'module/experiments/_affix';
+@import 'module/experiments/_membership';
```
Comment (Calanthe) [inline static/src/stylesheets/content.scss]: I know that I am a little bit picky right now, but I would rename the file to eg `_membership-test` os sth like that. The `membership` is too general.

File: static/src/stylesheets/module/experiments/_membership.scss [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+    // tablet: 28/32 = headline(5)
+    h2, h3 {
+        @include fs-headline(4);
+
+        @include mq(tablet) {
+            @include fs-headline(5);
+        }
+    }
+
+    // "How will it work?"
```
Comment (Calanthe) [inline static/src/stylesheets/module/experiments/_membership.scss]: Do you need those comments?

File: static/src/stylesheets/module/experiments/_membership.scss [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+
+    @include mq(phablet) {
+        background-position: right;
+        background-size: 50% auto;
+
+        padding-top: $gs-baseline * 2;
+        padding-bottom: $gs-baseline;
+    }
+
+    // "explained/explore/experience"
```
Comment (Calanthe) [inline static/src/stylesheets/module/experiments/_membership.scss]: Is it necessary?

File: static/src/stylesheets/module/experiments/_membership.scss [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+        }
+    }
+    @include mq($from: desktop) {
+        img.membership-test-tablet {
+            //padding-top: 88px;
+            padding-top: 60px;
+        }
+    }
+    //background-size: 50% auto;
+    //background-position: left;
```
Comment (Calanthe) [inline static/src/stylesheets/module/experiments/_membership.scss]: Remove those comments pls

File: static/src/stylesheets/module/experiments/_membership.scss [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+    //background-position: bottom;
+    background-position: 50% 160px;
+    background-size: 400px auto;
+
+    @include mq(tablet) {
+        //background-size: 50% auto;
+        background-position: 0 60px;
+        padding-bottom: 144px;
+        //background-size: 144px;
+        //background-position: left;
```
Comment (Calanthe) [inline static/src/stylesheets/module/experiments/_membership.scss]: Remove comments

**Comment by Calanthe** [comment]
I am concern about those images' sizes and dimensions. Especially that you load them on mobile breakpoint as well. Can you optimise or even remove some of them? Eg the pngs with transparency could be smaller.

**Comment by joelochlann** [comment]
@Calanthe Optimising the images is proving quite fiddly. Is it OK if I get this deployed (with the test turned off) so it can be looked over by some people, and address the image issues in a later PR?

**Pull request merged**